frontend/src/concepts/pipelines/content/DeletePipelineModalExpandableSection.tsx

```diff
@@ -58,7 +58,7 @@ const DeletePipelineModalExpandableSection: React.FC<DeletePipelineModalExpandab
           }
 
           return (
-            <ListItem key={resource.id} icon={icon}>
+            <ListItem key={resource.display_name + i} icon={icon}>
```

Comment:
Interesting -- that's inconvenient... I wonder if we can make a utility that reads properties and returns an id -- I don't like the idea of index because it leads to react slowness. 

if you use an integer in each key, React de-optimizes and rerenders the whole list each time -- instead of just the element that changed.